pom.xml

```diff
@@ -66,7 +66,6 @@
         <scalatest.version>3.2.9</scalatest.version>
         <scala.logging.version>3.9.4</scala.logging.version>
 
-        <ubirch-crypto.version>2.1.2</ubirch-crypto.version>
```

Comment:
I found it's not used.
Therefore, I deleted it to avoid that the dependency shows up when I searched dependencies of this lib